test/string-class-test.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+const assert = require('assert');
+require('../src/string-class');
+
+describe('Test for hasVowels() method', () => {
+  it('should return false if dfhgjl.hasVowels() was executed', () => {
```

Comment:
__Nitpick__
Test case descriptions are too similar to assertions.
_e.g "should return true for words with vowels"_
same applies to most of your test descriptions.